### PR TITLE
Arabicstories

### PR DESCRIPTION
**Pull request by Ako0**

### CODE REVIEW

File: stories/ar/29.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "language": "arabic",
+  "story": 29,
+  "title": "قرارك",
+  "sections": [
+    "من خلال هذه القصص، وقلت لكم عن الكثير من الناس الذين يعتقدون في الله. قلت لكم عن هابيل ونوح وإبراهيم وموسى وغيرهم. انهم جميعا اعترفوا بأنهم كانوا مذنبين. انهم يعتقدون انهم يستحقون الموت.",
+    "كما أعرب عن ثقته في المخلص الذي قال الله سيأتي. لذلك، غفر الله لهم خطاياهم وخلصهم من العقاب الأبدي.",
+    "كان هناك، وسوف يكون دائما طريقة واحدة فقط ليتم حفظها من الخطيئة، وفصل من الله والعقوبة الأبدية، عليك أن تؤمن مثل هابيل ونوح وإبراهيم وموسى أنك آثم الذي يستحق عقوبة الموت. عليك أن تؤمن المخلص، يسوع، الذي توفي بدلا منك، لذلك يمكن مغفورة لك خطاياك.",
+    "البعض منكم قد يعتقد، \"وكانت هذه قصة جيدة، ولكن أريد أن تفكر في ذلك وقررت لاحقا إذا أنا ذاهب إلى الاعتقاد يسوع هو منقذي.\" أريدك أن الاستماع إلى القصة وانا ذاهب الآن أن أقول لك .",
+    "يوم واحد، عندما كان يسوع ما زال هنا على الأرض، وقال الناس هذه القصة حول بحيرة النار. وقال: \"كان هناك رجل غني الذين يرتدون ملابس جميلة جدا. عاش في ترف كل يوم، ولكن عند بابه وضعت متسول اسمه لعازر.",
```
Comment (BrettBukowski) [inline stories/ar/29.json]: i can't tell for sure, but looks like the closing quote + comma are missing here?

File: stories/ar/29.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "قال الرجل المعاناة، \"أتوسل إليك، أرسل لعازر إلى منزل والدي لبلدي خمسة أشقاء. دعه تحذيرهم حتى لا تأتي إلى هذا المكان الرهيب. أجاب إبراهيم: \"إخوانكم لها كتابات موسى والأنبياء.",
+    "السماح لهم الاستماع إليهم. \"\" لا، والد إبراهيم، \"الرجل توسل،\" إذا كان شخص من بين الأموات يذهب إليهم يتوبون. \"",
+    "قال له إبراهيم: إذا إخوانكم لا يستمعون إلى موسى والأنبياء، فإنها لن تكون مقتنعا حتى إذا ارتفع شخص من بين الأموات \".",
+    "قال يسوع هذه القصة لتحذير لنا. نحن لا نعرف متى سيكون لدينا اليوم الأخير. نحن لا نعرف ما إذا كنا سوف تعيش غدا. يوم واحد، ونحن سوف يموت الجميع والحكم الوجه ثم بعد فوات الأوان أن نقرر ما إذا كنا سوف نرى في يسوع المخلص لدينا.",
+    "كل واحد منا يجب الاستماع إلى القصة من الله أن موسى والأنبياء وكتب أسفل. هذه هي القصة قلنا لك الآن، قصة المخلص.",
+    "كل واحد منا يجب أن تقرر في هذه الحياة إذا كنا سوف نرى في يسوع المخلص لدينا أم لا. بعد أن يموت، لا يمكننا تغيير الوضع لدينا أو مساعدة الآخرين في عائلتنا.",
+    "وقد أعطى الله لك هذا اليوم فرصة لتقرر ما إذا كنت سوف يؤمنون أن يسوع هو المخلص الخاص بك. ما هو قرارك؟ هل تعتقد فى المسيح؟ هل تعتقد أنه مات من أجل خطاياك وهذه هي الطريقة الوحيدة التي يمكن انقاذه؟",
```
Comment (BrettBukowski) [inline stories/ar/29.json]: and here?

File: stories/ar/chapters.js [hunk 3]
```diff
@@ -27,6 +27,8 @@ export default {
     require('./24.json'),
     require('./25.json'),
     require('./26.json'),
-    require('./27.json')
+    require('./27.json'),
+    require('./28.json'),
+    require('./29.json')
```
Comment (BrettBukowski) [inline stories/ar/chapters.js]: we might want to add a todo comment that we're still waiting on the official translations for these two chapters

File: stories/ar/29.json (hunk 1 above)
Comment (Ako0) [inline stories/ar/29.json]: not seeing what you're seeing, but i can tell you that the file is valid json and i can page through all the sections of the 29th story in arabic

**Pull request merged**